pkg/operator/status.go

```diff
@@ -385,6 +387,139 @@ func (optr *Operator) syncMetrics() error {
 	return nil
 }
 
+func (optr *Operator) syncClusterFleetEvaluation() error {
+	co, err := optr.fetchClusterOperator()
+	if err != nil {
+		return err
+	}
+	if co == nil {
+		return nil
+	}
+
+	unexpectedEvaluations, err := optr.generateClusterFleetEvaluations()
+	if err != nil {
+		return err
+	}
+
+	var coStatus configv1.ClusterOperatorStatusCondition
+	if len(unexpectedEvaluations) > 0 {
+		coStatus = configv1.ClusterOperatorStatusCondition{
```

Comment:
any reason (preference being a legit one) to not just 
```
return optr.updateStatus(co, configv1.ClusterOperatorStatusCondition{...})
```

instead of branching on assignment and returning below? could cut down a couple of lines and make it cleaner IMO.

Reply:
Got this fixed. Thanks!